packages/http-client/src/http-client-types.ts

```diff
@@ -88,10 +88,16 @@ interface KnownHttpResponseFields {
   request: any;
 }
 
+/**
+ * This interface is compatible with AxiosResponse.
+ */
 export interface HttpResponse extends KnownHttpResponseFields {
   [otherKey: string]: any;
 }
 
+/**
+ * Type of options used for e.g., [[executeHttpRequest]].
+ */
```

Comment:
[req] Can you also add a sentence what kind of options? It looks like those would be the only options when calling `executeHttpRequest`. I think we should give a bit more background.